travel-journal/index.md

```diff
@@ -0,0 +1,910 @@
+---
+title: 'Personal Travel Journal with Vue.js and Directus'
+description: 'This tutorial covers how to create a personal travel journal with Directus and Vue.js with authentication features'
+author:
+  name: 'Mercy bassey'
+  avatar_file_name: 'mercy-bassey.jpg'
+---
+
+## Introduction
+
+Vue.js, a progressive JavaScript framework known for its simplicity and flexibility, has gained widespread popularity among developers for building interactive user interfaces. In this tutorial, you we’ll build a personal travel journal application using Vue.js in conjunction with Directus, an open-source headless CMS. In this guide we’ll go through the process of implementing user authentication with Directus and demonstrate how to interact with a Directus collection using the Vue.js framework.
```

Comment:
```suggestion
In this tutorial, you we’ll build a personal travel journal application using Vue.js and Directus. We’ll go through the process of implementing user authentication with Directus and demonstrate how to interact with a Directus collection using the Vue.js framework.
```

I think opening with explaining what Vue is isn't strictly needed - readers should know this coming in. Even if not, you explain that it's a framework at the end of the paragraph. 

Other suggestions are just to make this more succinct.